### DISCUSSION TITLE
How do I specify Cabal flags?

### DISCUSSION THREAD

**Discussion by billksun** [srid/haskell-template#102] [Q&A] [answered]
How do I specify Cabal flags in my flake file?

**Comment by srid** [comment, marked as answer]
In `overrides`:

https://github.com/NixOS/nixpkgs/blob/b04b3345f16716f7289724286cdc8474d4758a41/pkgs/development/haskell-modules/lib/compose.nix#L174-L175

**Reply by srid** [reply]
Here's an example: https://github.com/nammayatri/common/blob/3ba4249bbd4bc81233502f504f5b8e0924d79911/nix/haskell.nix#L22-L24
